src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/ConnectionTab.php

```diff
@@ -22,13 +23,19 @@ class ConnectionTab extends AbstractTab
     use FormElementTrait;
 
     const DISCONNECT_ACTION = 'disconnect-action';
+    const CONNECT_ACTION = 'connect-action';
     const SAVE_ACTION = 'save-action';
 
+//    const INTEGRATIONS_HOST = 'https://integrations.storekeeper.software';
+    const INTEGRATIONS_HOST = 'https://junmar-accounts.code4.pizza';
```

Comment:
can you make it 
```
if( !defined('STOREKEEPER_WOOCOMMERCE_INTEGRATIONS') ) {
define('STOREKEEPER_WOOCOMMERCE_INTEGRATIONS', 'https://integrations.storekeeper.software');
}
```

 in `storekeeper-woocommerce-b2c.php`  when we can overwrite it in `wp-config.php` easily

Reply:
Done.